### DISCUSSION TITLE
Display Problem

### DISCUSSION THREAD

**Discussion by doubleus61** [tbnobody/OpenDTU#823] [General]
Hallo zusammen.
Ich habe gestern mit openDTU angefangen und finde das Projekt super. Tolle Leistung.

Da ich noch keine Panels habe, habe ich den ESP, das Funkmodul und das Display erstmal nur provisorisch zusammen gesteckt. Funktionier doweit ich es testen konnte problemlos, mit Wlan und LAN. Nur das Display zeigt an einem Rand unabhängig vom Drehwinkel ein beliebiges Muster von ca. 2 Pixeln. Das Display ist dieses:
[https://www.exp-tech.de/displays/oled/7343/waveshare-10444-1.3-inch-oled-a](url)
[https://www.waveshare.com/wiki/1.3inch_OLED_(A)](url)

Was könnte da schief laufen?

**Comment by knickohr** [comment]
Den falschen Displaytyp in den Settings ausgewählt :

Driver Chip: SH1106

ist nicht SSD1306 !

**Reply by doubleus61** [reply]
Danke für den Denkanstoß. Es war 1106 ausgewählt, aber die Datei "olimex_esp32_poe.json" ist fehlerhaft. Da waren beide Display als Typ 2 eingetragen.
